integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/SetAlarmNotifications.feature

```diff
@@ -27,41 +62,95 @@ Feature: SmartMetering Configuration - Set Alarm Notifications
-      | AlarmType_16         | FRAUD_ATTEMPT_M_BUS_CHANNEL_3         |
+      | AlarmType_16         | FRAUD_ATTEMPT_M_BUS_CHANNEL_1         |
       | AlarmTypeEnabled16   | false                                 |
-      | AlarmType_17         | FRAUD_ATTEMPT_M_BUS_CHANNEL_4         |
+      | AlarmType_17         | FRAUD_ATTEMPT_M_BUS_CHANNEL_2         |
       | AlarmTypeEnabled17   | false                                 |
-      | AlarmType_18         | NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_1 |
+      | AlarmType_18         | FRAUD_ATTEMPT_M_BUS_CHANNEL_3         |
       | AlarmTypeEnabled18   | false                                 |
-      | AlarmType_19         | NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_2 |
+      | AlarmType_19         | FRAUD_ATTEMPT_M_BUS_CHANNEL_4         |
       | AlarmTypeEnabled19   | false                                 |
-      | AlarmType_20         | NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_3 |
+      | AlarmType_20         | NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_1 |
       | AlarmTypeEnabled20   | false                                 |
-      | AlarmType_21         | NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_4 |
-      | AlarmTypeEnabled_21  | false                                 |
+      | AlarmType_21         | NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_2 |
+      | AlarmTypeEnabled21   | false                                 |
+      | AlarmType_22         | NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_3 |
+      | AlarmTypeEnabled22   | false                                 |
+      | AlarmType_23         | NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_4 |
+      | AlarmTypeEnabled23   | false                                 |
+      | AlarmType_24         | PHASE_OUTAGE_DETECTED_L1              |
+      | AlarmTypeEnabled24   | false                                 |
+      | AlarmType_25         | PHASE_OUTAGE_DETECTED_L2              |
+      | AlarmTypeEnabled25   | false                                 |
+      | AlarmType_26         | PHASE_OUTAGE_DETECTED_L3              |
+      | AlarmTypeEnabled26   | false                                 |
+      | AlarmType_27         | PHASE_OUTAGE_TEST_INDICATION          |
+      | AlarmTypeEnabled27   | false                                 |
     Then the specified alarm notifications should be set on the device
-      | DeviceIdentification | TEST1024000000001 |
+      | DeviceIdentification | TEST1024000000001                     |
+
+  @NightlyBuildOnly
+  Scenario: Set all alarm notifications disabled on a SMR 5.2 device
+    Given a dlms device
+      | DeviceIdentification | TEST1029000000001 |
+      | DeviceType           | SMART_METER_E     |
+      | Port                 | 1029              |
+      | Protocol             | SMR               |
+      | ProtocolVersion      | 5.2               |
+    When the set alarm notifications request is received
+      | DeviceIdentification | TEST1029000000001                     |
```

Comment:
Also add older alarm

Reply:
Added alarms in all registers for all protocol versions. 
Specifications say that alarm is registers not available on a device should be ignored